rxbindings/src/main/java/com/amplifyframework/rx/RxAuthBinding.java

```diff
@@ -162,6 +163,12 @@ public Single<AuthSession> fetchAuthSession() {
         return toSingle(delegate::fetchAuthSession);
     }
 
+    @Override
+    public Single<AuthSession> fetchAuthSession(AuthFetchSessionOptions options) {
```

Comment:
options should not be mandatory. Please add override with options.

Reply:
Looks like I missed `NonNull` annotation but there is already a fetchAuthSession overload where options are not required.